test/client.test.js

```diff
@@ -1839,100 +1839,6 @@ describe('ACC Client', function () {
         });
     });
 
-    // write unit test for client.fileUploader.uploadAemAsset method
-    describe("Tests for client.fileUploader.uploadAemAsset method", () => {
```

Comment:
Can you remove the formatting changes so that the actual changes can be seen?

Reply:
These are not due to formatting, I've moved test code with `fileUploader` class methods with following explaination:

`fileUploader` class has 3 methods now - `upload, uploadAemAsset, download`. earlier their test code was written at different places in file, I've organised them under 1 section for better readability as follows:

```
describe('File uploader', () => {
  describe('upload', () => {});
  describe('uploadAemAsset', () => {});
  describe('download', () => {});
});
```